sixMans/game.py

```diff
@@ -75,14 +77,14 @@ async def create_game_channels(self, category=None):
         # sync permissions on channel creation, and edit overwrites (@everyone) immediately after
         code = str(self.id)[-3:]
         self.textChannel = await guild.create_text_channel(
-            "{} {} {} Mans".format(code, self.queue.name, self.queue.maxSize), 
+            "{} {} {} Mans".format(code, self.queue.name, self.queue.maxSize),
```

Comment:
`.format` is an old convention we had. you can use f strings:
```py
f"{code} {self.queue.name} {self.queue.maxSize} Mans",
```

Reply:
I'm just removing blank spaces at end of line but I'll change to f-strings while I'm here lol